### ISSUE TITLE
Resizing preview shrinks the whole layout

### ISSUE THREAD

**Issue by wforts** [chinchang/web-maker#333]
## Meta info

- Is this a bug or suggestion?: **Bug**
- Version (click on help icon in footer): **3.4.2**
- Context - Web app, Chrome extension or both?: **Both**

Every time I resize the preview panel it grows this margin pixel by pixel on one side of the preview panel, and I can't get the layout back to full width. Is this a bug or am I doing something wrong?

https://i.imgur.com/TZMlVJQ.png
https://i.imgur.com/DAzYZ1j.png
https://i.imgur.com/f6fp8zH.png

**Comment by chinchang** [comment]
It's a bug, I am able to reproduce it. Will look into it.
Thanks for reporting 👍

**Comment by jarisgv** [comment]
Hello friends,
I had this bug when I started using Web Maker, and I solved that by changing the width in > .demo-side in  > style.css - I can't remember for sure because it's been a long time.
I just downloaded the latest release (3.5.1) and this bug still remains. The margin starts increasing as your code grows.
![image](https://user-images.githubusercontent.com/19142770/48008217-5f4e0b80-e100-11e8-8c8e-4a57f7e23a0a.png)

Best wishes.
:brazil:
